db/migrate/20150713143226_add_unique_to_operatingsystems_title.rb

```diff
@@ -0,0 +1,17 @@
+class AddUniqueToOperatingsystemsTitle < ActiveRecord::Migration
+  def up
+    duplicities = Operatingsystem.all.group_by(&:title).values.keep_if { |ary| ary.length > 1}
+    duplicities.each do |systems|
+      systems.each do |os|
+        os.description = os.fullname
```

Comment:
There /should/ be no duplicates of the fullname, but it's also not enforced at the DB level as far as I can tell, so I'm unconvinced that it's any better.

Is this bug theoretical or is there some data that could be added to the ticket so we can see precisely what situation occurred?